src/public/web.config.license

```diff
@@ -0,0 +1,3 @@
+SPDX-FileCopyrightText: 2024 Unsplash
```

Comment:
File is not copyright Unsplash (it's not a stock photo) and license should be updated to Apache 2.0